src/runtime_src/core/pcie/driver/linux/xocl/xocl_drm.h

```diff
@@ -24,8 +25,14 @@
 
 typedef void (*xocl_execbuf_callback)(unsigned long data, int error);
 
-#define IS_HOST_MEM(m_tag)	(!strncmp(m_tag, "HOST[0]", 7))
-#define IS_PLRAM(m_tag)		(!strncmp(m_tag, "PLRAM[", 6))
+#define IS_HOST_MEM(m_tag)				(!strncmp(m_tag, "HOST[0]", 7))
+#define IS_PLRAM(m_tag)					(!strncmp(m_tag, "PLRAM[", 6))
+#define IS_DDR(m_tag)					(!strncmp(m_tag, "DDR[", 4))
```

Comment:
The IS_DDR(m_tag) is incorrect because the legacy platform (u200) use different tag. Please revert this change.